addon/ng2/blueprints/ng2/files/src/client/main.ts

```diff
@@ -1,10 +1,13 @@
 import {bootstrap} from 'angular2/platform/browser';
 import {enableProdMode} from 'angular2/core';
+import {ROUTER_PROVIDERS} from 'angular2/router'
 import {environment} from './app/environment';
 import {<%= jsComponentName %>App} from './app/<%= htmlComponentName %>.component';
 
 if (environment.production) {
   enableProdMode();
 }
 
-bootstrap(<%= jsComponentName %>App);
+bootstrap(<%= jsComponentName %>App, [
```

Comment:
why add them here? the style guide advice (and from Igor) has been to not add providers in the main bootstrap, but instead to put them in the root app component